velox/core/SimpleFunctionMetadata.h

```diff
@@ -920,52 +942,66 @@ class UDFHolder {
 
   // Helper functions to handle void vs bool return type.
 
-  FOLLY_ALWAYS_INLINE bool callImpl(
+  FOLLY_ALWAYS_INLINE Status callImpl(
       typename Exec::template resolver<TReturn>::out_type& out,
+      bool& notNull,
       const typename Exec::template resolver<TArgs>::in_type&... args) {
     static_assert(udf_has_call);
-    if constexpr (udf_has_call_return_bool) {
+
+    if constexpr (udf_has_call_return_status) {
+      notNull = true;
```

Comment:
so the new call function cannot return nulls for now, and support for this will be added in a future PR?

Reply:
That's right.